jobs/FunctionTest/prepare_manifest.sh

```diff
@@ -38,27 +37,36 @@ wget_download(){
 
 }
 
+dlCommonFiles() {
+    dir=$1
+    mkdir -p ${dir} && cd ${dir}
+    if [ -n "${INTERNAL_HTTP_ZIP_FILE_URL}" ]; then
+        # use INTERNAL TEMP SOURCE
+        wget_download ${INTERNAL_HTTP_ZIP_FILE_URL}
+
+        unzip common.zip && mv common/* . && rm -rf common
+    else
+        # pull down index from bintray repo and parse files from index
+        wget_download --no-check-certificate https://dl.bintray.com/rackhd/binary/builds/ && \
+            exec  cat index.html |grep -o href=.*\"|sed 's/href=//' | sed 's/"//g' > files
+        for i in `cat ./files`; do
+            wget_download --no-check-certificate https://dl.bintray.com/rackhd/binary/builds/${i}
+        done
+        # attempt to pull down user specified static files
+        for i in ${HTTP_STATIC_FILES}; do
+            wget_download --no-check-certificate https://bintray.com/artifact/download/rackhd/binary/builds/${i}
+        done
+    fi
+}
 
 dlHttpFiles() {
-  dir=${WORKSPACE}/build-deps/on-http/static/http/common
-  mkdir -p ${dir} && cd ${dir}
-  if [ -n "${INTERNAL_HTTP_ZIP_FILE_URL}" ]; then
-    # use INTERNAL TEMP SOURCE
-    wget_download ${INTERNAL_HTTP_ZIP_FILE_URL}
+    dir=${WORKSPACE}/build-deps/on-http/static/http/common
+    dlCommonFiles ${dir}
+}
 
-    unzip common.zip && mv common/* . && rm -rf common
-  else
-    # pull down index from bintray repo and parse files from index
-    wget_download --no-check-certificate https://dl.bintray.com/rackhd/binary/builds/ && \
-        exec  cat index.html |grep -o href=.*\"|sed 's/href=//' | sed 's/"//g' > files
-    for i in `cat ./files`; do
-      wget_download --no-check-certificate https://dl.bintray.com/rackhd/binary/builds/${i}
-    done
-    # attempt to pull down user specified static files
-    for i in ${HTTP_STATIC_FILES}; do
-      wget_download --no-check-certificate https://bintray.com/artifact/download/rackhd/binary/builds/${i}
-    done
-  fi
+dlImageServiceCommonFiles() {
+    dir=${WORKSPACE}/build-deps/image-service/static/common
+    dlCommonFiles ${dir}
 }
```

Comment:
@changev  The dlImageServiceCommonFiles  is the same with dlHttpCommonFiles, instead of using the copy command, I brought about a common function for both use.